diff -Nrup a/configs/contentscanners/clamdscan.conf.in b/configs/contentscanners/clamdscan.conf.in
--- a/configs/contentscanners/clamdscan.conf.in	2020-01-27 09:24:39.000000000 +0100
+++ b/configs/contentscanners/clamdscan.conf.in	2020-02-17 07:27:19.907944906 +0100
@@ -1,7 +1,7 @@
 plugname = 'clamdscan'
 
 # edit this to match the location of your ClamD UNIX domain socket
-#clamdudsfile = '/var/run/clamav/clamd.sock'
+clamdudsfile = '@localstatedir@/clamav/clamd.socket'
 
 # If this string is set, the text it contains shall be removed from the
 # beginning of filenames when passing them to ClamD.
diff -Nrup a/configs/e2guardian.conf.in b/configs/e2guardian.conf.in
--- a/configs/e2guardian.conf.in	2020-01-27 09:24:39.000000000 +0100
+++ b/configs/e2guardian.conf.in	2020-02-17 07:28:58.854768662 +0100
@@ -105,7 +105,7 @@ logfileformat = 6
 # Log file location
 # 
 # Defines the log directory and filename.
-#loglocation = '@DGLOGLOCATION@/access.log'
+loglocation = '@DGLOGLOCATION@/access.log'
 
 # Defines optional request log path
 # This is for useful for debug purposes to log all requests before processing or setting filter group
@@ -425,7 +425,7 @@ maxcontentfilecachescansize = 20000
 # File cache dir
 # Where DG will download files to be scanned if too large for the
 # RAM cache.
-filecachedir = '/tmp'
+filecachedir = '@localstatedir@/e2guardian/tmp'
 
 # Delete file cache after user completes download
 # When a file gets save to temp it stays there until it is deleted.
@@ -587,7 +587,7 @@ httpworkers = 500
 # PID filename
 # 
 # Defines process id directory and filename.
-#pidfilename = '@DGPIDDIR@/e2guardian.pid'
+pidfilename = '@DGPIDDIR@/e2guardian.pid'
 
 # Disable daemoning
 # If enabled the process will not fork into the background.
@@ -614,8 +614,8 @@ loguseragent = off
 # Temp files created during virus scanning are given owner and group read
 
 # clamdscan, the two processes must run with either the same group or user ID.
-#daemonuser = '@DGPROXYUSER@'
-#daemongroup = '@DGPROXYGROUP@'
+daemonuser = '@DGPROXYUSER@'
+daemongroup = '@DGPROXYGROUP@'
 
 
 # Mail program
